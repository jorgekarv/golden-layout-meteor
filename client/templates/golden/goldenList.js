Template.goldenList.helpers({
  animals: function() {
    return Animals.find({},{ sort: {rank: 1}});
  }
});
