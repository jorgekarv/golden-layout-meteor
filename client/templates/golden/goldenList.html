<template name="goldenList">


  {{#each animals}}
    {{> animal}}
  {{else}}
    <h4>
      You haven't added any favorite animals, click the Add button!
    </h4>
  {{/each}}


</template>
